pkgs/applications/networking/browsers/misc/widevine-cdm.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "widevine-cdm";
+  version = "4.10.2449.0";
+
+  src = fetchzip {
+    url = "https://dl.google.com/widevine-cdm/${version}-linux-x64.zip";
+    sha256 = "sha256-f2kAkP+s3fB+krEZsiujEoI4oznkzSyaIB/CRJZWlXE=";
+    stripRoot = false;
+  };
+
+  installPhase = ''
+    install -vD manifest.json $out/share/google/chrome/WidevineCdm/manifest.json
```

Comment:
There isn't really a good reason to have all these subpaths. We can just dump it in lib.

I have a minimal version of this derivation here for example:
https://github.com/nrdxp/nrdos/blob/master/src/display/packages/widevine.nix

And it works just fine

Reply:
You are right. I copied the package `vivaldi-widevine` (see https://github.com/NixOS/nixpkgs/pull/194479/commits/5ef711b74148e6660b4ed7e97a2c2970adb3fcfd#diff-bde983dda613c89bbc071aec925d51f4fbc5b59a313024c42ec3d0606a188702) and in order to keep BC I should left these paths unmodified.

Maybe I should create a completely new package and revert the changes I have done on `vivaldi-widevine`? What do you think?